Source/ViewControllers/BrickViewController.swift

```diff
@@ -13,7 +13,7 @@ import UIKit
  Conform to this delegate on your preview view controller in order to enable UIKit Pop.
```

Comment:
I overlooked that UIViewControllerPreviewing is available in both iOS and tvOS, so I will make the adjustment to make these changes available in both shortly